### PR TITLE
Enable verbose logging for image-builder Ansible tasks

### PR DESCRIPTION
**Pull request by abhay-krishna**
Enable configuring verbosity for image-builder Ansible tasks through the `ansible-verbosity` flag.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh [hunk 1]
```diff
@@ -214,3 +214,17 @@ if [ "$IMAGE_FORMAT" = "ami" ]; then
     fi
 fi
 
+EKSA_ANSIBLE_VERBOSITY="${EKSA_ANSIBLE_VERBOSITY:-}"
+if [ -n "$EKSA_ANSIBLE_VERBOSITY" ]; then
+    PACKER_FILE_NAME="packer.json"
```
Comment (jaxesn) [inline projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh:219]: does this cover raw/nutanix as well?

**Review by jaxesn — COMMENTED** [review]
this looks pretty reasonable to me

File: projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh (hunk 1 above)
Comment (abhay-krishna) [reply projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh:219]: Yeah everything except OVA is packer.json

**Review by jaxesn — APPROVED** [review]

**Pull request merged**